### ISSUE TITLE
Clarify the relationship between ice-sip-sdp and RFC5245 + RFC6544

### ISSUE THREAD

**Issue by suhasHere** [suhasHere/ice-drafts#2]
It's not clear to me what the relationship is between this document and RFC 5245 (and, for that matter, 6544). Is this supposed to obsolete RFC 5245? That appears to be the intention. It's also not clear why we would obsolete 5245 without also incorporating 6544 -- it seems that doing so leaves the use of ICE TCP with SIP to be kind of in limbo.

Assuming the intention *is* to replace RFC 5245, the structure of this document seems to leave non-SIP uses of RFC3264 (e.g., WebRTC) stranded. If the intention is *not* to replace RFC 5245, the replication of all of the SDP syntax seems to be problematic: the the case of a conflict, which specification is intended to govern?

In short, I think the relationship between this document, the contemporary work (e.g., ICE-BIS), and the historical work on which it is based needs to be made much clearer in the document itself, and we need to have a clear picture about how to handle SDP for non-SIP uses of ICE (e.g., by splitting this document into an SDP part and a SIP part, similar to how we split 3264 from 3261).

**Comment by suhasHere** [comment]
1. This specification obsoletes RFC5245 as does the ICE-BIS specification

RFC6544
We agree that as of now RFC6544 is in limbo. So here is the plan. We have a editorial version of RFC6544bis at () that is updated to point to this specification and the ICE-BIS specification. 

Non SIP Use-cases
One suggestion is to split this specification into SDP usage for ICE spec and SIP Usage specs. This provides us the flexibility of getting WebRTC usage spec to be submitted later which refers to just the SDP Usage spec
